src/scss/grommet-core/_objects.box.scss

```diff
@@ -66,7 +66,6 @@
   min-height: 100vh;
   // min-height doesn't work for IE and vertical centering
   // https://connect.microsoft.com/IE/feedback/details/816293/ie11-flexbox-with-min-height-not-vertically-aligning-with-align-items-center
-  height: 100%;
```

Comment:
Have we verified that this doesn't cause problems for IE11 and vertically centering content, as indicated in the comment on line 68? Also, if we remove line 69, it would seem we should remove lines 67 and 68 as the comment must not apply anymore.